atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/paging/ColumnFetchMode.java

```diff
@@ -1,36 +0,0 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- *
- * Licensed under the BSD-3 License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://opensource.org/licenses/BSD-3-Clause
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.atlasdb.keyvalue.cassandra.paging;
-
-public final class ColumnFetchMode {
```

Comment:
🎉

Reply:
yeah this class was pretty weird. mainly the name